sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -193,6 +193,8 @@ case class CreateDataSourceTableAsSelectCommand(
       }
     }
 
+    CommandUtils.updateTableStats(sparkSession, table)
```

Comment:
It looks like a specific requirement.Maybe it should use 'ANALYZE TABLE <tablename> COMPUTE STATISTICS'.

Reply:
We should automatically collect it once `spark.sql.statistics.size.autoUpdate.enabled=true`. right?